lib/topology/manager.py

```diff
@@ -119,12 +120,15 @@ def load(self, dictmeta, inject=None):
                 ]
             }
 
-        See also the module :mod:`topology.parser`.
-
         :param dict dictmeta: The dictionary to load the topology from.
         :param dict inject: An attributes injection sub-dictionary as defined
          by :func:`parse_attribute_injection`.
         """
+        # Load the environment
+        self.environment = dictmeta.get('environment', OrderedDict())
+        if inject is not None and 'environment' in inject:
+            self.environment.update(inject['environment'])
```

Comment:
`Self.environment` is not passed in any way to the platform instance. Did you miss that?

Reply:
No we didn't miss it. I am waiting on feedback on #67 on how best to proceed passing the environment variables.

Reply:
option 1 is winning the race, currently only @sergiosmcr is against :P

Reply:
You are wrong. I agree with Option 1. In fact, it is already implemented in pynml and pyszn. However there seem to be bugs in pynml. See CI.